packages/parser/test/test.ts

```diff
@@ -23,14 +23,29 @@ describe('can parse all Less stylesheets', () => {
 });
 
 describe('should throw parsing errors', () => {
+  // TODO: There should be no errors!
```

Comment:
To reiterate my comment from before, this SHOULD have parsing errors in the future, so errors.length will need to be at least 1 depending on the test.